bin/cij_plotter

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python
```

Comment:
Is it on purpose that this is implemented in a single file, unlike the runner/reporter/analyser?